mainwindow.cpp

```diff
@@ -298,17 +307,14 @@ void MainWindow::on_pushButton_WordlistFileBrowse_clicked()
 void MainWindow::on_listWidgetTabs_itemSelectionChanged()
```

Comment:
Let's get rid of these on_ slots(done via the Designer "Got to slot") and use explicit connect().